### PR TITLE
doc: update documentation to reflect that driver does not support blue/green deployments

### PR DESCRIPTION
**Pull request by aaronchung-bitquill**
### Additional Reviewers

@sergiyv-improving @karenc-bq @aaron-congo @joyc-bq @brunos-bq

### CODE REVIEW

**Review by joyc-bq — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -625,6 +625,10 @@ public class AwsIamAuthenticationSample {
 }
 ```
 
+### Amazon RDS Blue/Green Deployments
```
Comment (karenc-bq) [inline README.md:628]: I think this should be in the Known Issues section. Can you confirm with Matthew.

Comment (aaronchung-bitquill) [reply README.md:628]: Good call out. I'll check with him.

Comment (aaronchung-bitquill) [reply README.md]: Updated to address this.

**Review by sergiyvamz — APPROVED** [review]

**Pull request merged**